modules/home/theme/gruvbox/default.nix

```diff
@@ -151,6 +152,16 @@ in {
           };
         };
 
+      services.polybar.config."colors" = {
+        background = "#282828";
+        background-alt = "#689d68";
+        foreground = "#ebdbb2";
+        foreground-alt = "#ebdbb2";
+        primary = "#689d68";
+        secondary = "#1d2021";
+        alert = "#cc241d";
```

Comment:
@rissson can you please revise these to match what we have declared in the let statement? Those colors are extracted directly from the gruvbox theme.

Reply:
Right now, this let statement is only for i3bar. This change should be in an other PR refractoring the way themes are handled. We should have a let statement at the beggining of the config file. Maybe even change the colors based on the chosen theme instead of one redundant file per theme to allow for easy add of themes